cmd/node/README.md

```diff
@@ -26,29 +26,30 @@ List of supported CLI flags is listed below.
   -r, --role string                    role this node will have in the Blockless protocol (head or worker) (default "worker")
   -c, --concurrency uint               maximum number of requests node will process in parallel (default 10)
       --boot-nodes strings             list of addresses that this node will connect to on startup, in multiaddr format
       --workspace string               directory that the node can use for file storage
-      --attributes                     node should try to load its attribute data from IPFS
-      --peer-db string                 path to the database used for persisting peer data (default "peer-db")
-      --function-db string             path to the database used for persisting function data (default "function-db")
+      --load-attributes                node should try to load its attribute data from IPFS
       --topics strings                 topics node should subscribe to
+      --db string                      path to the database used for persisting peer and function data
   -l, --log-level string               log level to use (default "info")
   -a, --address string                 address that the b7s host will use (default "0.0.0.0")
   -p, --port uint                      port that the b7s host will use
       --private-key string             private key that the b7s host will use
+      --dialback-address string        external address that the b7s host will advertise
+      --dialback-port uint             external port that the b7s host will advertise
   -w, --websocket                      should the node use websocket protocol for communication
       --websocket-port uint            port to use for websocket connections
-      --dialback-address string        external address that the b7s host will advertise (default "0.0.0.0")
-      --dialback-port uint             external port that the b7s host will advertise
       --websocket-dialback-port uint   external port that the b7s host will advertise for websocket connections
+      --purge-dialback-peers           purge peers known from previous runs
+      --rest-api string                address where the head node REST API will listen on
       --runtime-path string            Blockless Runtime location (used by the worker node)
-      --runtime-cli string             runtime CLI name (used by the worker node) (default "bls-runtime")
-      --cpu-percentage-limit float     amount of CPU time allowed for Blockless Functions in the 0-1 range, 1 being unlimited (default 1)
+      --runtime-cli string             runtime CLI name (used by the worker node)
+      --cpu-percentage-limit float     amount of CPU time allowed for Blockless Functions in the 0-1 range, 1 being unlimited
       --memory-limit int               memory limit (kB) for Blockless Functions
-      --rest-api string                address where the head node REST API will listen on
+      --purge-functions                purge installed functions
```

Comment:
I don't know how I feel about these being flags. I kinda feel (and I know we have this no where else at the moment) like these should be one time positional arguments? 
`b7s purge functions` `b7s purge peers`

Thoughts?

Reply:
I'm open to a discussion! Why don't you like it as flags? (not going either way, just curious).
We could also remove it as a flag completely and it would be available as a config file and environment variable, or is that also something you don't want..?

However, I do really dislike the idea of this being a special case with positional arguments.

Positional arguments have been something I was thinking about for a while now. However, 99% of the time we want to do `node=>run` and I think polluting it with a single other thing would be a waste.

However, now we do have [a few other things](https://github.com/blocklessnetwork/b7s/tree/main/cmd) we do.

We could switch to it being a tool like
```
# run b7s node like we do now
b7s run --config worker.yaml

# generate keys
b7s keys generate --output /var/tmp/keys

# peer actions
b7s peers list
b7s peers purge

# function actions
b7s functions list --space # show how much disk space installed functions take
b7s function install <cid>

# setup cgroup that we use for the resource limits
b7s limits bootstrap 

# node docs
b7s docs # or something else, I don't have a good idea right now
```

On the other hand, we could just create something like a control panel (`b7s-cmd`, `b7s-cmder` or `b7s-ctrl`, I dunno) and have all our tools and utils live inside it. I think something like that would be tremendously useful for introspection and testing. We could add support to install function asynchronously, async execution, merge in the headless worker node execution tool and a bunch of other stuff.

WDYT?